pkg/bpf/map_linux.go

```diff
@@ -884,6 +884,31 @@ func (m *Map) Delete(key MapKey) error {
 	return err
 }
 
+// DeleteIgnoreMetricErrorCount deletes the map entry without incrementing the
+// metric error count for the given key.
+func (m *Map) DeleteIgnoreMetricErrorCount(key MapKey, ignoreMetricErrorCount bool) error {
```

Comment:
IMHO, this method name reads as "this method deletes an ignore metric error counter".

Reply:
Should we call it something like `SilentDelete()` or something because it silences metrics which could be used to figure out what it's doing?